app/views/user.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import viewsets
+
+from ..models import User
+from ..serializers import UserSerializer
+
+
+class UserViewSet(viewsets.ModelViewSet):
```

Comment:
I don't know if you want to use a viewSet here. They don't implement the regular API CRUD operations like get and post

Reply:
I was able to use it to create and get users though. I'm not sure I understand, please explain? It's specifically because I was trying to use the user_id (not drf's default id) for retrieve/put/patch/delete operations that I even used viewset, also because I'm unable to properly write the logic to use user_id as the lookup id.

Reply:
I'll have to look into this also.

Reply:
Okay.